modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -179,8 +180,6 @@
 			gap: var(--n-tabs-icon-gap);
 			border-width: var(--n-tabs-border-width);
 			position: relative;
-			cursor: pointer;
```

Comment:
The button has a pointer by default, but was overridden by the all: unset styling.